install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -246,6 +246,7 @@ type UsageConfig struct {
 	Enabled                          bool               `json:"enabled"`
 	Schedule                         string             `json:"schedule"`
 	BillInstancesAfter               *time.Time         `json:"billInstancesAfter"`
+	DefaultSpendingLimit             map[string]int32   `json:"defaultSpendingLimit"`
```

Comment:
Instead of using a `map`, my preference would be for this to use the [DefaultSpendingLimit](https://github.com/gitpod-io/gitpod/blob/05c183e208aad2e57dd1cc0af85db1df9879554b/components/usage/pkg/db/cost_center.go#L41) struct.

As this is in the `experimental` section, I'll approve this on the basis that it's not for public usage, but it would keep with the consistency of using structs in the configuration